src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -113,392 +63,106 @@ const ReviewStep = () => {
     }
   });
 
-  const handleTabClick = (event, tabIndex) => {
-    setActiveTabKey(tabIndex);
-  };
+  const onToggle = (isExpanded) => setIsExpanded(isExpanded);
 
   return (
     <>
-      <Text>
-        Review the information and click &quot;Create image&quot; to create the
-        image using the following criteria.
-      </Text>
-      <DescriptionList isCompact>
-        <DescriptionListGroup>
-          {getState()?.values?.['image-name'] && (
-            <>
-              <DescriptionListTerm>Image name</DescriptionListTerm>
-              <DescriptionListDescription>
-                {getState()?.values?.['image-name']}
-              </DescriptionListDescription>
-            </>
-          )}
-          <DescriptionListTerm>Release</DescriptionListTerm>
-          <DescriptionListDescription>
-            {RELEASES.get(getState()?.values?.release)}
-          </DescriptionListDescription>
-        </DescriptionListGroup>
-      </DescriptionList>
-      <Tabs
-        isFilled
-        activeKey={activeTabKey}
-        onSelect={handleTabClick}
-        className="pf-u-w-75"
+      <ExpandableSection
+        toggleContent={'Image output'}
+        onToggle={onToggle}
+        isExpanded={isExpanded['image-output']}
```

Comment:
I'm not 100% sure about this. `isExpanded` is a boolean, so `isExpanded['image-output']` should return undefined... right? 

The expandable sections work fine when you actually run this in the browser and in the tests, though! Which is kind of surprising.

Still, I worry that this could lead to bugs in the future. Even though it works, we probably shouldn't be treating a boolean like an object... 

Maybe the solution is to give each expandable section its own useState hook?